components/root/root.jsx

```diff
@@ -447,6 +451,7 @@ export default class Root extends React.PureComponent {
                     <CompassThemeProvider theme={this.props.theme}>
                         <ModalController/>
                         <GlobalHeader/>
+                        {this.props.isUserFirstAdmin && !this.props.isMobile && this.props.dismissChecklist && <TaskList/>}
```

Comment:
```suggestion
                        {this.props.isUserFirstAdmin && !this.props.isMobile && !this.props.dismissChecklist && <TaskList/>}
```
as this value will not be there in preferences table, Although I think I can take care of this in my Pr

Reply:
missed it, sorry bout that guys :)